.github/workflows/develop-branch.yml

```diff
@@ -47,5 +47,12 @@ jobs:
         env:
           CF_USERNAME: ${{ secrets.CF_DEV_DEPLOYMENT_SVC }}
           CF_PASSWORD: ${{ secrets.CF_DEV_DEPLOYMENT_SVC_PWD }}
+      - name: Init Application
+        run: devops/scripts/cf-app-init.sh
+      - name: Configure Env Vars (secrets)
+        run: |
+          cf set-env $APP_NAME REACT_APP_ECMPS_API_KEY $REACT_APP_ECMPS_API_KEY
+        env:
+          REACT_APP_ECMPS_API_KEY: ${{ secrets.ECMPS_API_KEY }}
```

Comment:
need to add the environment step just under the runs-on: ubuntu-latest step above and the api key needs to be set to the non production secret